jupinx/theme/minimal/templates/latex.tpl

```diff
@@ -0,0 +1,304 @@
+            ((( custom_add_prompt(output.data['text/latex'] | wrap_text(88) | ansi2latex, cell, 'Out', 'outcolor', 'latex') )))
+        ((* else -*))
+            ((( " " )))
+            ((( draw_prompt(cell, 'Out', 'outcolor','') )))((( super() )))
+        ((*- endif -*))
+    ((*- endfor -*))
+((* endblock execute_result *))
+
+% Display stream ouput with coloring
+((* block stream *))
+\begin{Verbatim}[commandchars=\\\{\}, fontsize=\footnotesize]
+    ((( output.text | wrap_text(86) | escape_latex | ansi2latex )))
+\end{Verbatim}
+%((* endblock stream *))
+
+%==============================================================================
+% Define macro custom_add_prompt() (derived from add_prompt() macro in style_ipython.tplx)
+%==============================================================================
+
+((* macro custom_add_prompt(text, cell, prompt, prompt_color, type) -*))
+    ((*- if cell.execution_count is defined -*))
+    ((*- set execution_count = "" ~ (cell.execution_count | replace(None, " ")) -*))
+    ((*- else -*))
+    ((*- set execution_count = "" -*))
+    ((*- endif -*))
+    ((*- set indention =  " " * (execution_count | length + 7) -*))
+\begin{Verbatim}[mathescape, commandchars=\\\{\}, fontsize=\small, xleftmargin=-3.9em]
+((( text.replace('$$','').replace('$','') | add_prompts(first='{\color{' ~ prompt_color ~ '}' ~ prompt ~ '[{\\color{' ~ prompt_color ~ '}' ~ execution_count ~ '}]:} ', cont=indention) )))
+\end{Verbatim}
+((*- endmacro *))
+
+%==============================================================================
+% Support Macros
+%==============================================================================
+
+% Name: draw_prompt
+% Purpose: Renders an output/input prompt
+((* macro draw_prompt(cell, prompt, prompt_color, extra_space) -*))
+    ((*- if cell.execution_count is defined -*))
+    ((*- set execution_count = "" ~ (cell.execution_count | replace(None, " ")) -*))
+    ((*- else -*))((*- set execution_count = " " -*))((*- endif *))
+
+    ((*- if (resources.global_content_filter.include_output_prompt and prompt == 'Out')
+         or (resources.global_content_filter.include_input_prompt  and prompt == 'In' ) *))
+\prompt{(((prompt)))}{(((prompt_color)))}{(((execution_count)))}{(((extra_space)))}
+    ((*- endif -*))
+((*- endmacro *))
+
+
+%==============================================================================
+% Bibliography
+%==============================================================================
+
+% Insert citations in markdown as e.g.
+%    <cite data-cite="DevoretS2013">[DevoretS2013]</cite>
+% requires file references.bib in current directory (or the file set as "bib" in the latex_metadata)
+
+
+((* block bibliography *))
+((*- if nb.metadata.get("latex_metadata", {}).get("bib_include", ""): -*))
+% Add a bibliography block to the postdoc
+\bibliographystyle{plain}
+\bibliography{((( nb.metadata.get("latex_metadata", {}).get("bib", "quant-econ") )))}
```

Comment:
@AakashGfude this looks like it has some hard coding in it for the bibfile?

Reply:
@AakashGfude can we have the template use the bibfile specified in the `conf.py`? How do you get that variable?

Reply:
HI @mmcky, that `quant-econ` written in that line, is just a default value, when no bib file is specified. At present, we are including the bib file from the variable specified in `conf.py` itself.